### DISCUSSION TITLE
Only detects 2 of 3 fans on RTX 2080 Super ?

### DISCUSSION THREAD

**Discussion by SporeProductions** [Rem0o/FanControl.Releases#2275] [Q&A] [answered]
Is it normal that FanControl detected only 2 of the 3 fans my graphics card has? (Asus RTZ 2080 Super (8/24GB) )

**Comment by Rem0o** [comment, marked as answer]
Yes. Your card runs these 3 fans on 2 fan channels.  2 fans are on one channel.
